extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php

```diff
@@ -114,11 +117,11 @@ static public function onBeforeParserMakeImageLinkObjOptions( $parser, $title, &
 			$alt = implode( '.', array_slice( explode( '.', $text ), 0, -1 ) ); // lop off text after the ultimate dot (e.g. JPG)
 			$parts[] = "alt={$alt}";
 		}
-		
+
 		return true;
-		
```

Comment:
My IDE removes unnecessary whitespace, so these can be ignored or viewed with `?w=1`